specification/azureactivedirectory/resource-manager/Microsoft.Aadiam/stable/2020-03-01/privateEndpointConnections.json

```diff
@@ -0,0 +1,534 @@
+          }
+        },
+        "description": "Gets the specified private endpoint connection associated with the given policy.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/policyNameParameter"
+          },
+          {
+            "$ref": "#/parameters/privateEndpointConnectionName"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Private endpoint connection successfully returned.",
+            "schema": {
+              "$ref": "#/definitions/PrivateEndpointConnection"
+            }
+          },
+          "default": {
+            "description": "Aadiam error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "patch": {
+        "description": "Updates an existing Private Endpoint Connection's tags. To update other fields use the Put method.",
+        "operationId": "PrivateEndpointConnections_UpdateTags",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/policyNameParameter"
+          },
+          {
+            "$ref": "#/parameters/privateEndpointConnectionName"
+          },
+          {
+            "name": "privateLinkConnectionTags",
+            "description": "Updated tag information to set into the PrivateLinkConnection instance.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/TagsResource"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Updating the AzureAD PrivateLinkConnection's tags was successful. PrivateLinkConnection tags are updated and returned with the rest of the PrivateLinkConnection's object properties.",
+            "schema": {
+              "$ref": "#/definitions/PrivateEndpointConnection"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "AadiamUpdatePrivateEndpointConnections": {
+            "$ref": "./examples/AzureADPrivateEndpointConnectionsUpdate.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ErrorResponse": {
```

Comment:
Please consider referencing common-types: https://github.com/Azure/azure-rest-api-specs/blob/1d28da6ddb09d18ef055e36f57c781e0f76eb3ed/specification/common-types/resource-management/v2/types.json#L322

Reply:
Updating this in my last PR was a breaking change from 2020/03/01-preview, since it's missing the default field I believe.

Reply:
It's okay not to use the common type if it causes validation error. (I wonder why that happens though..)

Reply:
Not sure, here's the output from the previous PR I tried to add default and type (which is in the common type I believe).

https://github.com/Azure/azure-rest-api-specs/pull/14882/checks?sha=3c55d237f2af33213ce3164aa6bc8aa937549f8a